src/Apps/CollectorProfile/Routes/Saves2/Components/CreateNewListModal/AddArtworksModalContent.tsx

```diff
@@ -139,6 +139,7 @@ const AddArtworksModalContentPaginationContainer = createPaginationContainer(
           sort: { type: CollectionArtworkSorts, defaultValue: POSITION_DESC }
         ) {
         collection(id: "saved-artwork") {
+          artworksCount(onlyVisible: true)
           artworksConnection(first: $first, after: $after, sort: $sort)
             @connection(key: "AddArtworksModalContent_artworksConnection") {
             totalCount
```

Comment:
**question**: do we need query `totalCount` if `artworksCount` is used?

Reply:
Not needed indeed. I thought that it's required for pagination logic